src/bindings/python/tests/test_graph/test_preprocess.py

```diff
@@ -393,7 +393,7 @@ def test_graph_preprocess_steps(algorithm, color_format1, color_format2, is_fail
             "Gather",
             "Interpolate",
         ]
-        assert len(model_operators) == 16
+        assert len(model_operators) == 15
```

Comment:
Why did you change the number of operations?

Reply:
The number of operations is lower, because the new Interpolate-11 has single merged input for scales and sizes instead of two separate inputs as Interpolate-4, so there is one less Constant op.